### ISSUE TITLE
Etablere nasjonalt ValueSet for personidentifikatorer

### ISSUE THREAD

**Issue by oaassv** [HL7Norway/basisprofiler-r4#118]
Både VKP og NILAR har behov for å benytte er verdisett med de ulike personidentifkatorene - fnr, hnr, dnr og fhr. HL7 Norge etablerer et nytt verdisett som skal kunne benyttes av alle som har behov for å referere til disse 4.

**Comment by rockphotog** [comment]
Basisprofilene (typ Patient etc.) skal kun ha én måte å identifisere identifikasjonstyper for personer. Dette er løst per dags dato. 

Behovet til VKP/NILAR bør beskrives bedre her før jeg kan komme med et bedre råd (implementert vs område vs basis, etc.)

Jeg antar at verdisett fra Volven er utgangspunkt for behovet. Et verdisett som beskrevet slik kan publiseres som en tjeneste på https://github.com/HL7Norway/kodeverk

**Comment by thomiz** [comment]
* [Subject identifier](https://hl7norway.github.io/VkpObservation/currentbuild/ValueSet-vkp-subject-identifiers.valueset.html)
* [Performer identifier](https://hl7norway.github.io/VkpObservation/currentbuild/ValueSet-vkp-performer-identifiers.valueset.html)

**Comment by rockphotog** [comment]
Identifiserte krav: [Subject|Performer]/identifier/system skal være **uri**. Beskrivelse: *The namespace for the identifier value*. 

Ergo må et eventuell binding vise til et sett (ValueSet) med et antall gyldige namespace (uri...).

ValueSet er i hovedsak knyttet til implementasjonsguide, så det viktigste er å lage et CodeSystem som inneholder namespace for alle aktuelle identifiseringsmåter. 

Ett for org. og ett for personer, eller et felles CodeSystem? Kanskje best med det første så det er lett å lage VS?

**Comment by thomiz** [comment]
I dette tilfellet angir vi jo identifikatoren for namespacet, for eksempel **urn:oid:2.16.578.1.12.4.1.4.1**  

Så det er vel akkurat disse verdiene vi ønsker å kontrollere blir brukt med disse kodeverkene: 
* [Kodeverk for subject identifier](https://hl7norway.github.io/VkpObservation/currentbuild/CodeSystem-vkp-performer-identifiers.codesystem.html)
* [Kodeverk for performer identifier](https://hl7norway.github.io/VkpObservation/currentbuild/CodeSystem-vkp-subject-identifiers.codesystem.html)

Men akkurat kodeverket for subject identifier trenger flere verdier i no-basis, siden kodeverket for Vkp er svært begrenset (bare fødselsnummeret). Så kan VkpSubjectidentifier verdisettet innskrenke dette til bare FNR verdien.

Enig @rockphotog ?

**Comment by rockphotog** [comment]
Ja. Jeg har begynt her, vel og merke under kodeverk-repoet, men det kan alltids flyttes. 

https://github.com/HL7Norway/kodeverk/blob/identifikator-namespace/input/fsh/codesystems/no-kodeverk-identifikator-namespace.fsh
